terrautils/sensors.py

```diff
@@ -118,75 +121,94 @@
               "fixed_metadata_datasetid": "TBD"
         },
 
-        'rgb_fullfield': {
+        'fullfield': {
+            'display': 'Full Field Stitched Mosaics',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{date}{opts}.tif',
         },
 
-        'flirIrCamera': {
+        'hyperspectral2nc_vnir': {
+            'display': 'VNIR Hyperspectral NetCDFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
-            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
+        },
+
+        'hyperspectral2nc_swir': {
+            'display': 'SWIR Hyperspectral NetCDFs',
+            'template': '{base}/{station}/Level_1/'
+                        '{sensor}/{date}/{time}/{filename}',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
         },
 
         'stereoTop_geotiff': {
+            'display': 'stereoTop GeoTIFFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
         },
 
         'stereoTop_canopyCover': {
+            'display': 'stereoTop Canopy Cover',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
         },
 
         'texture_analysis': {
+            'display': 'stereoTop R Texture Analysis',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.csv',
```

Comment:
here are the contents of a texture CSV file:
```
roi;area;edges;dgci.-0.1;dgci.0;dgci.0.1;dgci.0.2;dgci.0.3;dgci.0.4;dgci.0.5;dgci.0.6;dgci.0.7;dgci.0.8;dgci.0.9;dgci.1;dgci.1.1;dgci.1.2;dgci.1.3;dgci.1.4;dgci.1.5;dgci.1.6;dgci.1.7;dgci.1.8;dgci.1.9;dgci.2;dgci.2.1;dgci.2.2;dgci.2.3;dgci.2.4;dgci.2.5;dgci.2.6;dgci.2.7;dgci.2.8;dgci.2.9;m.cx;m.cy;m.majoraxis;m.eccentricity;m.theta;s.area;s.perimeter;s.radius.mean;s.radius.sd;s.radius.min;s.radius.max
1;8147712;1064733;0;0;1;1036;12654;51407;151118;701880;2445980;3203387;1238079;300676;23768;9364;1408;325;1156;1134;115;584;15;202;2331;1092;0;0;0;0;0;0;0;1236.5;1648.5;3805.89279933106;0.661437858208952;1.5707963267949;8147712;11532;1646.77211050119;242.29069883532;1235.50010117361;2059.30000242801
```
I don't think it is something we send to BETY, so I was retaining them.